README.md

```diff
@@ -1,2 +1,30 @@
-# factory-trytond
-Factory Boy - Trytond integration
+# Factory-trytond
+
+Factory-trytond is a [factory_boy](https://factoryboy.readthedocs.io/en/latest/introduction.html) extension developed to work with [Tryton ERP](https://www.tryton.org/). We can create our own ERP's model factories to do some testing or to populate our databases with some sample data.
+
+## How does it work?
+
+To use Factory-trytond your factory class only has to inherit the TrytonFactory:
+
+    import factory_trytond
+
+    class SaleFactory(factory_trytond.TrytonFactory):
```

Comment:
I think this stage is unnecessary. Anyone reading this doc will know how to inherit a class and what it means